dias/analyzers/sensitivity_analyzer.py

```diff
@@ -152,6 +158,15 @@ def setup(self):
         cursor.execute(CREATE_DB_TABLE)
         self.data_index.commit()
 
+        # Add a data metric for sensitivity.
+        self.sens = self.add_data_metric(
+            "Thermal_noise_RMS",
```

Comment:
```suggestion
            "average_sensitivity",
```

The name should match the documentation (alternatively call it `thermal_noise_rms` in the documentation).